synapseclient/client.py

```diff
@@ -7533,6 +7709,55 @@ def _uploadCsv(
 
         Returns:
             [UploadToTableResult](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/table/UploadToTableResult.html)
+
+        Example: Using this function (DEPRECATED)
+            Uploading a CSV file to an existing table:
+
+                response = syn._uploadCsv(
+                    filepath='/path/to/table.csv',
+                    schema='syn123'
+                )
+
+        Example: Migration to new method
+            &nbsp;
+
+            ```python
+            import asyncio
+            from synapseclient import Synapse
+            from synapseclient.models import Table
+            from synapseclient.models.mixins.table_components import CsvTableDescriptor
```

Comment:
If users should be using a particular class can this be exposed at the `synapseclient.models` level as well? So that you could do:

`from synapseclient.models import Table, CsvTableDescriptor`

Reply:
Got it. I added them to models/__init__.py, so the import should be working as expected now.